client/src/components/TestSuite/TestSuiteTree/styles.tsx

```diff
@@ -36,15 +36,15 @@ export default makeStyles((theme: Theme) => ({
     marginRight: '5px',
   },
   testSuiteTreePanel: {
-    width: '400px',
+    width: '300px',
     overflowX: 'hidden',
+    marginLeft: '-10px',
   },
   treeRoot: {
     '& $labelText': {
-      fontWeight: 600,
-    },
-    '& .MuiTreeItem-iconContainer': {
-      display: 'none',
+      textTransform: 'uppercase',
+      fontWeight: 'bold',
     },
+    paddingLeft: '25px !important',
```

Comment:
This is already default behavior?

Reply:
Yeah this is weird.  I was fiddling with where the root of the tree goes vs the other nodes, and where it is relative to the far left side of the page.  This all can be rethought, but it was the quickest way for me to get it to look ok.